cmd/run.go

```diff
@@ -297,6 +302,10 @@ func runInteractive(ctx context.Context, flags RunFlags) error {
 		return err
 	}
 
+	if flags.LaunchStudio {
```

Comment:
As discussed, would rather have this happen _after_ run compeltes

Reply:
+1 to that

Reply:
Yeah makes it easier to test for now, will do that in the next PR